packages/block-editor/src/components/block-list-appender/style.scss

```diff
@@ -14,11 +14,6 @@
 		margin: $grid-unit-10 0;
 	}
 
-	// Add an explicit left margin of zero and auto right margin to work in horizontal
-	// flex containers. Without it, a "space-between"-like effect from two auto margins
-	// will cause the black plus to sit in the center of what space is left.
-	margin: 0 auto 0 0;
```

Comment:
I think we still need _some_ margin. Can you try the code from https://github.com/WordPress/gutenberg/pull/32574/files#diff-c01c18b571a5d47e64bd8d17a9d00ef9a67f1ceb2877b8ea967ce4a0e23b6b22R15?

Reply:
I updated with this `$grid-unit-10` works well.
The important fix is removing the auto, so having some margin is good :+1: